external/footer.php

```diff
@@ -7,15 +7,15 @@
     $isAjax = true;
 }
 
-if (extension_loaded('xhprof') && $_xhprof['doprofile'] === true) {
+if ( $_xhprof['ext_name'] && $_xhprof['doprofile'] === true) {
     $profiler_namespace = $_xhprof['namespace'];  // namespace for your application
-    $xhprof_data = xhprof_disable();
+    $xhprof_data = call_user_func($_xhprof['ext_name']."_disable");
```

Comment:
Well, in a performance point of view (not a readability) , only after 10k or so iterations is when you notice single quotes being faster than double quotes.
But yeah, single are faster, will fix